packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -317,11 +317,13 @@ class RunCommand extends RunCommandBase {
     }
 
     DateTime appStartedTime;
-    // Sync completer so the completing agent attaching to the resident doesn't 
-    // need to know about analytics. 
+    // Sync completer so the completing agent attaching to the resident doesn't
+    // need to know about analytics.
     //
     // Do not add more operations to the future.
     final Completer<Null> appStartedTimeRecorder = new Completer<Null>.sync();
+    // Callback hookups' Futures are uninteresting.
```

Comment:
Unfortunately the linter doesn't allow that kind of syntax :(
https://github.com/dart-lang/linter/issues/706

Has to be on different lines. One hack is to do `// ignore: unawaited_futures, [our custom text]` but I'm worried it might bring up random linter bugs since the follow on text might be interpreted as linter instructions.